### PR TITLE
TWiRx 8

### PR DESCRIPTION
**Pull request by Ticki**
'omebody need'a review t' gramma' tho'

### CODE REVIEW

File: content/news/this-week-in-redox-8.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- [@jackpot51](https://github.com/jackpot51) have improved multithreading support and the syslog.
+
+- And lots of other small changes.
+
+
+# What does it look like?
+
+Changes have not affected the appereance a lot:
+
+![start](https://raw.githubusercontent.com/redox-os/redox/master/img/screenshots/screen.png)
```
Comment (hauleth) [inline content/news/this-week-in-redox-8.md]: Again :P

Comment (Ticki) [inline content/news/this-week-in-redox-8.md]: What's wrong?

Comment (hauleth) [inline content/news/this-week-in-redox-8.md]: Link to master. Should be in canonical form (should link to specific commit).

Comment (Ticki) [inline content/news/this-week-in-redox-8.md]: It's fixed.

**Pull request merged**